kits/job-posting-notion-sync/README.md

```
@@ -0,0 +1,198 @@
+### 4. Set Your Notion Database ID
+
+Open [`flows/job.ts`](./flows/job.ts) and find the `notionNode_1` values block. Replace the placeholder with your actual Notion database ID:
+
+```ts
+// flows/job.ts  ~line 239
+"databaseId": "REPLACE_WITH_YOUR_NOTION_DATABASE_ID",
+```
+
+Your database ID is the part of the Notion page URL between the last `/` and the `?`:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)